ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/mapper/PaymentIntentionMapper.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.cmc.ccd.domain.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPaymentOption;
+import uk.gov.hmcts.cmc.ccd.domain.CCDPaymentSchedule;
+import uk.gov.hmcts.cmc.ccd.domain.response.CCDPaymentIntention;
+import uk.gov.hmcts.cmc.ccd.mapper.Mapper;
+import uk.gov.hmcts.cmc.domain.models.PaymentOption;
+import uk.gov.hmcts.cmc.domain.models.RepaymentPlan;
+import uk.gov.hmcts.cmc.domain.models.ccj.PaymentSchedule;
+import uk.gov.hmcts.cmc.domain.models.response.PaymentIntention;
+
```

Comment:
more than one line blank left.

Reply:
removed. I suggest in the future you add any of these rules to CheckStyle it will save us both a lot of time and effort. we just can't keep making formatting rules on the fly.